### PR TITLE
update author

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**JIRA**
https://issues.jboss.org/browse/RHMAP-10943

**Steps**
update package.json

@grdryn @PhilipGough

### CODE REVIEW

**Comment by grdryn** [comment]
@laurafitzgerald there are package.json files in the individual components also (these are more important than the top level one really). Could you check if they need to be updated? I think they might.

Also, let's wait until the job is configured and has run against this before merging :)

File: package.json [hunk 1]
```diff
@@ -9,6 +9,14 @@
...
+    "type": "git",
+    "url": "git://github.com/feedhenry/fh-messaging.git"
+  },
+    "bugs": {
+    "url": "https://github.com/feedhenry/fh-config/issues"
+  },
+  "author": "FeedHenry",
   "dependencies": {
     "moment": "^2.10.6"
   },
```
Comment (PhilipGough) [inline package.json]: @laurafitzgerald Can we remove the caret ^ here and update moment to 2.15.2 please to match our shrinkwrap

File: package.json [hunk 2]
```diff
@@ -9,6 +9,14 @@
   "scripts": {
     "test": "make test"
   },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/feedhenry/fh-messaging.git"
+  },
+    "bugs": {
+    "url": "https://github.com/feedhenry/fh-config/issues"
```
Comment (laurafitzgerald) [inline package.json]: yes

File: package.json [hunk 3]
```diff
@@ -23,11 +31,6 @@
   "engines": {
     "node": ">=0.6"
   },
```
Comment (PhilipGough) [inline package.json]: @grdryn Is this engine correct or should it be node 4.4 ?

**Comment by PhilipGough** [comment]
@laurafitzgerald A couple of comments here, maybe @grdryn  can chime in to see if he agrees. Id like to see these added to the initial commit as opposed to committing on top if it

**Comment by laurafitzgerald** [comment]
@PhilipGough sure will take a look and sqaush the two commits when done

File: package.json (hunk 3 above)
Comment (grdryn) [inline package.json]: @PhilipGough AIUI, this package.json doesn't actually get used at all. It's the ones in the individual components that are important

File: package.json (hunk 1 above)
Comment (grdryn) [inline package.json]: See my other comment, may not be worth it if this package.json is never used for an `npm install`.

**Comment by grdryn** [comment]
Retest this please.

**Comment by PhilipGough** [comment]
Retest this please.

File: fh-messaging/package.json [hunk 4]
```diff
@@ -4,10 +4,12 @@
   "version": "3.0.3-BUILD-NUMBER",
   "repository": {
     "type": "git",
-    "url": "dberesford@amz1svn1.feedhenry.com:/var/git/fh-messaging.git"
+    "url": "git://github.com/feedhenry/fh-messaging.git"
   },
-  "author": "Damian Beresford",
-  "license": "Apache-2.0",
+    "bugs": {
```
Comment (grdryn) [inline fh-messaging/package.json]: No biggie, right, but I think this line might be indented more than expected, is it?

File: fh-metrics/package.json [hunk 5]
```diff
@@ -4,10 +4,12 @@
   "version": "3.0.3-BUILD-NUMBER",
   "repository": {
     "type": "git",
-    "url": "/git/fh-messaging.git"
+    "url": "git://github.com/feedhenry/fh-messaging.git"
   },
-  "author": "Martin Murphy",
-  "license": "Apache-2.0",
+    "bugs": {
```
Comment (grdryn) [inline fh-metrics/package.json]: Same here, this line looks like it's in a step too far.

File: package.json [hunk 6]
```diff
@@ -9,6 +9,14 @@
   "scripts": {
     "test": "make test"
   },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/feedhenry/fh-messaging.git"
+  },
+    "bugs": {
```
Comment (grdryn) [inline package.json]: Same here, maybe indented a step too far.

**Pull request merged**